<?xml version="1.0"?>
<launch>

	<!-- serial -->
	<node name="serial_asio" pkg="slam_car" type="serial_asio" 
	>
	</node>
	
	<!-- public odometry message and tf -->
	<node name="odom_tf_pub" pkg="slam_car" type="odom_tf_pub" 
	>
	</node>
	
	<!-- base controller and tf -->
	<node name="base_controller" pkg="slam_car" type="base_controller" 
	output="screen">
	</node>
	
	<!-- diff drive controller -->
	<include file="$(find slam_car)/launch/slave_diff_drive_controller.launch">
	</include>
	
	
</launch>
